.github/workflows/dedupe.yaml

```diff
@@ -0,0 +1,54 @@
+name: Deduplicate Dependabot
```

Comment:
Even with this being split out in a separate workflow, I still do not think this is something we should add.

If this is an issue, that should be documented and reported to GitHub.
We should not add this level of workarounds, that to me, at least clearly show that something is wrong with how it works.

Reply:
Agree with @ludeeus on this.

As an alternative could be: Don't use dependabot for this task.
An alternative could be, for example, Renovate. That is a competitor to dependabot that can actually dedupe (if configured as such).

../Frenck

Reply:
Renovate seems like a nice option

Reply:
I agree Renovate is probably the better option for HA.  In addition to being able to dedupe, it also handles monorepos much better, which has also been annoying from dependabot.

Closing this, but for posterity bringing it up to date and also added a simple change which addresses @ludeeus's concern about dependabot losing its rebase capability when committing the dedupe.

I'll submit a request to install Renovate and would be happy to work on the config PR.